src/organisms/tables/ComparisonTable/index.js

```diff
@@ -23,9 +25,7 @@ const ComparisonTable = (props) => {
   return (
     <div className={tableClass}>
       {
-        React.Children.map(children, child =>
-          React.isValidElement(child) ? React.cloneElement(child, { expandable, expanded, tableItems }) : child
-        )
+        React.Children.map(children, (child) => React.isValidElement(child) ? React.cloneElement(child, { expandable, expanded, tableItems }) : child)
```

Comment:
brackets can keep the multi-line arrow functions

Reply:
changed